gnome/gobject-introspection/Portfile

```diff
@@ -79,6 +94,12 @@ configure.args      -Ddoctool=enabled \
 # should not require modification.
 build.env-append    CC=${configure.cc}
 
+foreach arch ${muniversal.architectures} {
+    # There seem to be parts of the build system that do not have the `-arch` flag.
+    build.env.${arch}-append    "CFLAGS=${configure.cflags}   [option configure.ld_archflags.${arch}]"
+    build.env.${arch}-append    "LDFLAGS=${configure.ldflags} [option configure.cc_archflags.${arch}]"
```

Comment:
these look backwards? (LDFLAGS should have configure configure.ld_archflags)

Reply:
not that it makes any difference, one hopes...

Reply:
Thank you for noticing.
The order should now be fixed.